### ISSUE TITLE
Propagate ALTER TABLE foo ENABLE TRIGGER to workers

### ISSUE THREAD

**Issue by hanefi** [citusdata/citus#2924]
`ALTER TABLE foo ENABLE TRIGGER` queries are not propagated to workers. Current workaround is using `enable_ddl_propagation` flag to manually propagate this query.

```
set citus.enable_ddl_propagation TO OFF;
ALTER TABLE foo ENABLE REPLICA TRIGGER bar;
SELECT run_command_on_placements($$$
ALTER TABLE foo ENABLE REPLICA TRIGGER bar;
$$$);
set citus.enable_ddl_propagation TO ON;
```

**Comment by surprisetalk** [comment]
This worked for us (with some modifications)!

`SET citus.enable_ddl_propagation TO OFF` only sets the config on the current session on the current machine. So we added `citus.enable_ddl_propagation = 'OFF'` to each worker's `postgresql.conf` and then executed `SELECT pg_reload_conf()` on the workers. After that, we could run the following on our master:
``` sql
SELECT run_command_on_placements
( 'example_distributed_table'
, 'ALTER TABLE %1$I ENABLE REPLICA TRIGGER example_trigger'
)
;
```

**Comment by aykut-bozkurt** [comment]
Fixed by #4143

An example execution:

```sql
CREATE TABLE employees(
   id int,
   first_name VARCHAR(40),
   last_name VARCHAR(40)
);

SELECT create_distributed_table('employees', 'id');

CREATE TABLE employee_audits(
   id int,
   last_name VARCHAR(40),
   changed_on TIMESTAMP(6)
);

SELECT create_distributed_table('employee_audits', 'id');

SET citus.enable_unsafe_triggers TO true;

CREATE OR REPLACE FUNCTION log_last_name_changes()
  RETURNS TRIGGER
  AS
$$
BEGIN
        IF NEW.last_name <> OLD.last_name THEN
                 INSERT INTO employee_audits(id,last_name,changed_on)
                 VALUES(OLD.id,OLD.last_name,now());
        END IF;

        RETURN NEW;
END;
$$ LANGUAGE plpgsql;

CREATE TRIGGER last_name_changes
  BEFORE UPDATE ON employees
  FOR EACH ROW
  EXECUTE PROCEDURE log_last_name_changes();

ALTER TABLE employees ENABLE REPLICA TRIGGER last_name_changes;
```
